apps/src/templates/sectionProgress/SectionProgressToggle.jsx

```diff
@@ -2,34 +2,40 @@ import React, { PropTypes } from 'react';
 import ToggleGroup from '../ToggleGroup';
 import color from "@cdo/apps/util/color";
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import { connect } from 'react-redux';
+import {setCurrentView, ViewType} from './sectionProgressRedux';
 
 /**
  * A toggle that provides a way to switch between detail and summary views of
  * the progress a section of students have made in a course. Teacher view.
  */
 class SectionProgrsesToggle extends React.Component {
   static propTypes = {
-    isSummaryView: PropTypes.bool.isRequired,
-    toggleView: PropTypes.func.isRequired,
+    currentView: PropTypes.string.isRequired,
```

Comment:
`currentView` is in redux and `SectionProgressToggle` is connected, but you're not getting currentView from redux.  Is there a particular reason?